### PR TITLE
Fix typos in site pages and _config.yml

### PR DESCRIPTION
**Pull request by Systemmanic**
Fixed some typos and grammar for the website.

Fixed grammar in comments of config.

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -80,6 +80,6 @@ Be sure you have added `node_modules` to your Sass's [load_paths](http://stackov
 ## What's next
 
 - [Documentations](docs): Design philosophy and tutorial
-- [Reference](reference): All the Mobi.css styles in one page
```
Comment (cht8687) [inline source/index.md:83]: 'in' or 'on' here? I feel 'in' is slightly better....

**Review by cht8687 — APPROVED** [review]

File: source/index.md (hunk 1 above)
Comment (Systemmanic) [reply source/index.md:83]: Yeah I guess it makes more sense in this context.

Could change it to something like 'See the Mobi.css styles in action' or 'Examples of the Mobi.css styles' or a combination.

Something that directs the scanning eye of developers who want to see examples fast. I hate having to hunt for snippets.

Comment (cht8687) [reply source/index.md:83]: @dkanejs  agree.~ we can merge this.

**Comment by cht8687** [comment]
@xcatliu  we can merge this. I don't have write access to this repo I think.

**Comment by xcatliu** [comment]
Thanks @dkanejs and @cht8687 !

**Pull request merged**